Formula/spot.rb

```diff
@@ -0,0 +1,15 @@
+class Spot < Formula
+  desc ": a platform for LTL and ω-automata manipulation"
+  homepage "https://spot.lrde.epita.fr/"
+  url "https://www.lrde.epita.fr/dload/spot/spot-2.9.8.tar.gz"
+  sha256 "b7f404bb90a335a5914384ecc3fc3a2021ff22c57ee97a40c07bb2ab40e20cf9"
+  license "GPL-3.0-or-later"
+
+  def install
+    system "./configure", *std_configure_args, "--disable-silent-rules"
+  end
```

Comment:
Seems to be missing the `make` step.

Reply:
Yes sorry, I have added the necessary build step. Thank you for the quick feedback.